<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QDialog" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>356</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>356</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tela de Ações</string>
  </property>
  <widget class="QTableWidget" name="tablePlayList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>66</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>277</number>
   </attribute>
   <column>
    <property name="text">
     <string>Minhas PlayList</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>42</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PlayList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="criarPlay">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar PlayList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMusic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Musica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delMusic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Deletar Musica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="execPlayList">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Executar Musica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchMusic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar Musica</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>3</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableMusic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>388</number>
   </attribute>
   <column>
    <property name="text">
     <string>Musicas</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
